spring-cloud-app-broker-docs/src/docs/asciidoc/service-instances.adoc

```diff
@@ -29,6 +29,7 @@ spring:
             *health-check: http*
             *health-check-http-endpoint: /health*
             *health-check-timeout: 180*
+            *api-polling-timeout: 300*
```

Comment:
See https://github.com/spring-cloud/spring-cloud-app-broker/pull/289/files#r351238763

Reply:
resolving to avoid duplicate conversations about the property naming